src/assets/accessories/eye_patch/graphics.json

```diff
@@ -35,25 +45,27 @@
 			"y": 0,
 			"width": 1000,
 			"height": 1500,
-			"type": "mesh",
-			"name": "patch_back",
-			"priority": "BELOW_BACK_HAIR",
+			"type": "autoMesh",
+			"name": "back_mesh",
 			"points": "static",
-			"mirror": 0,
-			"colorizationKey": "patch",
-			"image": {
-				"image": "",
-				"overrides": [
-					{
-						"image": "eye_patch_back.png",
-						"condition": [
-							[
-								{
-									"view": "back"
-								}
-							]
-						]
-					}
+			"automeshTemplate": "belowBody",
```

Comment:
Hi @SandrinePDR, could you look at this? It might be a bug.
Before change the back part would be visible above the hair, but this will prevent that.